test/Api.Test/Controllers/CiphersControllerTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Security.Claims;
+using AutoFixture.Xunit2;
+using Bit.Api.Controllers;
+using Bit.Api.Models.Request;
+using Bit.Core.Context;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Settings;
+using Core.Models.Data;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.Controllers;
+
+public class CiphersControllerTests : IDisposable
+{
+    private readonly GlobalSettings _globalSettings;
+    private readonly ICurrentContext _currentContext;
+    private readonly ICipherRepository _cipherRepository;
+    private readonly ICollectionCipherRepository _collectionCipherRepository;
+    private readonly ICipherService _cipherService;
+    private readonly IUserService _userService;
+    private readonly ILogger<CiphersController> _logger;
+    private readonly IAttachmentStorageService _attachmentStorageService;
+    private readonly IProviderService _providerService;
+    private readonly CiphersController _ciphersController;
+
+    public CiphersControllerTests()
+    {
+        _currentContext = Substitute.For<ICurrentContext>();
+        _globalSettings = Substitute.For<GlobalSettings>();
+        _cipherRepository = Substitute.For<ICipherRepository>();
+        _collectionCipherRepository = Substitute.For<ICollectionCipherRepository>();
+        _userService = Substitute.For<IUserService>();
+        _cipherService = Substitute.For<ICipherService>();
+        _logger = Substitute.For<ILogger<CiphersController>>();
+        _attachmentStorageService = Substitute.For<IAttachmentStorageService>();
+        _providerService = Substitute.For<IProviderService>();
+
+        _ciphersController = new CiphersController(_cipherRepository, _collectionCipherRepository, _cipherService,
+            _userService, _attachmentStorageService, _providerService, _currentContext, _logger, _globalSettings);
```

Comment:
Check out other controller test files. You should be able to decorate this class with 
```csharp
[ControllerCustomize(typeof(CiphersController))]
[SutProviderCustomize]
```

then instead of `[AutoData]`, decorate the test with `[BitAutoData]` and expect a parameter of type `SutProvider<CiphersController> sutProvider`